src/main.rs

```diff
@@ -238,7 +275,18 @@ fn main() -> Result<()> {
                             },
                             vec![wasm].into_iter().chain(conf),
```

Comment:
Couldn't we do the config switch on this line instead to reduce duplication?

Reply:
done